pandas/tests/extension/base/missing.py

```diff
@@ -88,6 +88,11 @@ def test_fillna_limit_backfill(self, data_missing):
         tm.assert_series_equal(result, expected)
 
     def test_fillna_no_op_returns_copy(self, data):
+        if data.dtype.kind == "c":
+            pytest.skip(
+                f"no cython implementation of backfill(ndarray[{data.dtype.name}_t],"
+                f"ndarray[{data.dtype.name}_t], int64_t) in libs/algos.pxd"
```

Comment:
if you must skip/xfail, the place to do it is in test_numpy, not in the base tests

Reply:
Done.